xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1588,6 +1588,17 @@ void CDVDDemuxFFmpeg::DisposeStreams()
   m_parsers.clear();
 }
 
+bool CDVDDemuxFFmpeg::isFpsNonsense(const AVRational &frameRate)
```

Comment:
Please place in an anonymous namespace at the start of the file. This doesn't need to be a class method. I'm also not sure but it may be able to be `constexpr`.

```suggestion
constexpr bool CDVDDemuxFFmpeg::isFpsNonsense(const AVRational &frameRate)
```